docs/quick-start.md

```diff
@@ -36,6 +36,18 @@ The second command will download and run a prebuilt AsterixDB docker container f
 After it finishes, you should see the messages as shown in the following screenshot:
 ![docker][docker]
 
+After you have installed the docker container for the first time, you may restart it later using the following command:
```

Comment:
I still have a feeling that this part is just a "good to know" information, which doesn't serve the purpose of "quick-start". we can also tell them how to start and stop the AsterixDB, but I don't know if it's this page's task. 

I think the developer need to know this command, it should go to wiki?